### PR TITLE
Add Failure details into comment

### LINKED ISSUE #133: Option to add failed tests as comment
When PRs are huge, finding annotations is a chore.
If a summary of failed tests are added as a comment like coverage, it would be much easier to review.

I can raise the PR if the feature can be added.

### PR DESCRIPTION
**Pull request by sidharthv96**
Fixes #133

Demo PR: https://github.com/sidharthv96/jest-coverage-report-action-test/pull/1

### CODE REVIEW

**Comment by sidharthv96** [comment]
@ArtiomTr I've replaced Webpack with esbuild, hope that's fine.
The build time went from ~50sec to ~1sec.

File: src/format/getFormattedFailures.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export const getFailureDetails = (report: JsonReport): string =>
+    report.testResults &&
+    report.testResults.some(({ message }) => message.length > 0)
+        ? '\n```bash\n' +
+          report.testResults
+              ?.map(({ message }) => stripAnsi(message))
+              .join('```\n---\n```bash\n') +
+          '```'
+        : '';
```
Comment (ArtiomTr) [inline src/format/getFormattedFailures.ts:13]: Could you take a look at #132? This PR describes and fixes some issues with such error format.

**Review by ArtiomTr — CHANGES_REQUESTED** [review]
Everything seems to be good, just one small remark about the formatting of failed tests.

Also, I think I have merged your PRs in the wrong order, so these changes are now conflicting with the current Refactor branch changes. Could you please resolve them?

File: src/format/getFormattedFailures.ts (hunk 1 above)
Comment (sidharthv96) [reply src/format/getFormattedFailures.ts]: Done in https://github.com/ArtiomTr/jest-coverage-report-action/pull/137/commits/0478397a7b51219c415389670ec49c97714a06c1

File: src/format/annotations/formatFailedTestsAnnotations.ts [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
 export const formatFailedTestsAnnotations = (
-    jsonReport: JsonReport,
+    runReport: TestRunReport,
     annotations: Array<Annotation>
 ): CreateCheckOptions => ({
     ...context.repo,
     status: 'completed',
     head_sha: context.payload.pull_request?.head.sha ?? context.sha,
-    conclusion: 'success',
+    conclusion: 'failure',
```
Comment (sidharthv96) [inline src/format/annotations/formatFailedTestsAnnotations.ts:21]: @ArtiomTr This assumption is correct right?
If the tests failed, then the check should be marked failure.

**Review by ArtiomTr — APPROVED** [review]
Perfect ✨

File: src/format/annotations/formatFailedTestsAnnotations.ts (hunk 2 above)
Comment (ArtiomTr) [reply src/format/annotations/formatFailedTestsAnnotations.ts:21]: Yes, I think that this is right behavior. Thank you for your contribution!

**Pull request merged**